cypress/integration/document_level_monitor_spec.js

```diff
@@ -336,6 +336,38 @@ describe('DocumentLevelMonitor', () => {
         // Confirm we can see the new trigger
         cy.contains(newTriggerName);
       });
+
+      it('with only 1 index', () => {
+        // This test ensures the bug in this issue has been fixed
+        // https://github.com/opensearch-project/alerting-dashboards-plugin/issues/258
+
+        // Creating the test monitor
+        cy.createMonitor(sampleDocumentLevelMonitor);
+        cy.reload();
+
+        // Confirm the created monitor can be seen
+        cy.contains(SAMPLE_DOCUMENT_LEVEL_MONITOR);
+
+        // Select the monitor
+        cy.get('a').contains(SAMPLE_DOCUMENT_LEVEL_MONITOR).click({ force: true });
+
+        // Click Edit button
+        cy.contains('Edit').click({ force: true });
+
+        // Click on the Index field and type in multiple index names to replicate the bug
+        cy.get('#index')
+          .click({ force: true })
+          .type(`indexName1{enter}indexName2{enter}`, { force: true })
+          .trigger('blur', { force: true });
+
+        // Confirm Index field only contains the expected text
+        cy.get('[data-test-subj="indicesComboBox"]').should('not.have.text', TESTING_INDEX);
+        cy.get('[data-test-subj="indicesComboBox"]').should('not.have.text', 'indexName1');
+        cy.get('[data-test-subj="indicesComboBox"]').should('have.text', 'indexName2');
```

Comment:
Does this mean it will only take the last index inputted?

Reply:
Correct. My thought process is that if a user is typing into the combo box, they likely want to overwrite whatever is currently there with the new input.